visualizations/bloch-sphere/static/style.css

```
@@ -0,0 +1,143 @@
+  color-scheme: light;
+  --paper: var(--atlas-viz-background);
+  --panel: var(--atlas-viz-panel);
+  --panel-subtle: var(--atlas-viz-panel-subtle);
+  --ink: var(--atlas-viz-text);
+  --muted: var(--atlas-viz-muted);
+  --line: var(--atlas-viz-border);
+  --accent: var(--atlas-viz-accent);
+  --focus: var(--atlas-viz-focus-ring);
+  --state-a: #1f77b4;
+  --warning: #a33a50;
+  font-family: Inter, "Helvetica Neue", Arial, "Hiragino Sans", "Yu Gothic UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)